src/nvim/if_cscope.c

```diff
@@ -32,13 +30,17 @@
 #include "nvim/quickfix.h"
 #include "nvim/strings.h"
 #include "nvim/tag.h"
+#include "nvim/window.h"
+
+#include <sys/stat.h>
+#include <sys/types.h>
 #if defined(UNIX)
-# include <sys/wait.h>
+#include <sys/wait.h>
 #endif
 #include "nvim/if_cscope_defs.h"
 
 #ifdef INCLUDE_GENERATED_DECLARATIONS
-# include "if_cscope.c.generated.h"
+#include "if_cscope.c.generated.h"
```

Comment:
this does not match the convention used elsewhere. macros are indented by scope (and the "scope" here is the `#ifdef`).

Reply:
@justinmk, thanks for the heads-up! I've found the setting to tell clang-format to handle this
https://github.com/neovim/neovim/blob/0cf56905d5328b09e8da2ee19435af563bcdbcb3/.clang-format#L48

I've been trying to update the [rules in .clang-format](https://github.com/kylo252/neovim/blob/8b03456290d9d17824a0f45ac83692404d3bfbd6/.clang-format) to help get consistent headers sorting in this PR, according to the pattern used in other places in the code, (uncrustify's rules aren't consistent either sadly)

But clang-format is re-using the same indent width, so I still ended up needing to use both tools on different lines... :(